src/routers/handlers/individualPscList/individualPscList.ts

```diff
@@ -0,0 +1,39 @@
+import { Request, Response } from "express";
+import {
+    BaseViewData,
+    GenericHandler,
+    ViewModel
+} from "../generic";
+import logger from "../../../lib/Logger";
+import { PrefixedUrls } from "../../../constants";
+import { selectLang, getLocalesService, getLocaleInfo } from "../../../utils/localise";
+
+export class IndividualPscListHandler extends GenericHandler<BaseViewData> {
+
+    private static templatePath = "router_views/individualPscList/individualPscList";
+
+    public getViewData (req: Request): BaseViewData {
+
+        const baseViewData = super.getViewData(req);
+        const lang = selectLang(req.query.lang);
+        const locales = getLocalesService();
+
+        return {
+            ...baseViewData,
+            ...getLocaleInfo(locales, lang),
+            title: "Which PSC are you providing verification details for?",
```

Comment:
I think this needs to be removed as Simon and we learnt this am.  And the title set in the template and pulled from the language files